storage_service/storage_service/settings/base.py

```diff
@@ -428,6 +429,52 @@ def is_true(env_str):
 
     ALLOW_USER_EDITS = False
 
+
+OIDC_AUTHENTICATION = is_true(environ.get("SS_OIDC_AUTHENTICATION", ""))
+if OIDC_AUTHENTICATION:
+    ALLOW_USER_EDITS = False
+
+    AUTHENTICATION_BACKENDS += ["common.backends.CustomOIDCBackend"]
+    LOGIN_EXEMPT_URLS.append(r"^oidc")
+    INSTALLED_APPS += ["mozilla_django_oidc"]
+
+
+    # AUTH_SERVER = 'https://login.microsoftonline.com/common/v2.0/'
+    OIDC_RP_CLIENT_ID = environ.get("OIDC_RP_CLIENT_ID", "")
+    OIDC_RP_CLIENT_SECRET = environ.get("OIDC_RP_CLIENT_SECRET", "")
+
+    OIDC_OP_AUTHORIZATION_ENDPOINT = ""
+    OIDC_OP_TOKEN_ENDPOINT = ""
+    OIDC_OP_USER_ENDPOINT = ""
+    OIDC_OP_JWKS_ENDPOINT = ""
+
+    AZURE_TENANT_ID = environ.get("AZURE_TENANT_ID", "")
+    if AZURE_TENANT_ID:
```

Comment:
What if you don't set an AZURE_TENANT_ID? Can you use OpenID functionality?

Reply:
Good point. I guess it should attempt to read those from the environment instead.